react/src/navigation/navigator.tsx

```diff
@@ -410,6 +413,15 @@ async function loadPageDescriptor(newDescriptor: PageDescriptor, settings: Setti
             let quizDescriptor: QuizDescriptor
             let todayName: string
             switch (newDescriptor.mode) {
+                case 'custom':
+                    const custom = JSON.parse(base64Gunzip(newDescriptor.quizContent ?? '')) as CustomQuizContent
```

Comment:
Please use zod schemas here for better error handling

Reply:
is there a way to do this in a way that doesn't require rewriting this whole thing into zod types?
![image](https://github.com/user-attachments/assets/1f9a011b-9c25-4403-b1f9-0095318108a0)

Reply:
.... no... probably not worth it I guess for this use case